pypeit/spectrographs/magellan_mage.py

```diff
@@ -46,8 +46,8 @@ def __init__(self):
                             numamplifiers   = 1,
                             gain            = 1.02, # depends on the readout
                             ronoise         = 2.9, # depends on the readout
-                            datasec         = '[1:2048,1:1024]',      # complementary to oscansec
-                            oscansec        = '[2049:2176,1:1024]'    # as taken from the header
+                            datasec         = '[1:1024, 1:2048]',
```

Comment:
Not specific to Magellan MagE, but do we always need to specify datasec and oscansec now.  I.e., can we ever read it from the file header?

Reply:
I fear for many/most instruments that we do..